code/game/objects/items/storage/storage.dm

```diff
@@ -62,9 +62,10 @@
 	return dat
 
 /obj/item/storage/compile_monkey_icon()
+	var/identity = [type]_[icon_state]
```

Comment:
This should be `"[type]_[icon_state]"`; currently missing quotation marks

Reply:
Also might want to leave a comment for clarity to make sure why this change is the way it is, for example: "Makes it so that you can use multiple icon states for a piece of clothing"